common/djangoapps/django_comment_common/migrations/0003_auto_20161107_0954.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+from django.db import migrations, models
+
+def add_default_enable(apps, schema_editor):
+    ForumsConfig = apps.get_model("django_comment_common", "ForumsConfig")
+    settings_count = ForumsConfig.objects.count()
+    if settings_count is 0:
+        settings = ForumsConfig(enabled=True)
```

Comment:
Instead of defaulting to being enabled, can you look for (`getattr`) the enable forums flag in Django settings, and use that as the default when no `ForumsConfig` entries exist?